Formula/libvdpau.rb

```diff
@@ -17,25 +17,21 @@ class Libvdpau < Formula
     sha256 mojave:        "59980ec6bf90b676354ddda5e3c93a6240c4564d1c01aa35b1f1aa804d7b949a"
   end
 
-  depends_on "pkg-config" => [:build, :test]
+  depends_on "meson" => [:build]
+  depends_on "ninja" => [:build]
+  depends_on "pkg-config" => [:build]
```

Comment:
```suggestion
  depends_on "meson" => :build
  depends_on "ninja" => :build
  depends_on "pkg-config" => [:build, :test]
```

I think you still need `pkg-config` for the test.